src/app/projects/projects.service.spec.ts

```diff
@@ -1,15 +1,131 @@
-import { TestBed } from '@angular/core/testing';
+import { async, TestBed } from '@angular/core/testing';
 
 import { ProjectsService } from './projects.service';
-import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { environment } from "../../environments/environment";
```

Comment:
There should be quotes instead of double quotes.

Reply:
done